packages/paste-core/components/alert-dialog/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {matchers} from 'jest-emotion';
+import {render, screen} from '@testing-library/react';
+// @ts-ignore typescript doesn't like js imports
+import axe from '../../../../../.jest/axe-helper';
+import {AlertDialogForNotifying, AlertDialogDestructive, AlertDialogExample} from '../stories/index.stories';
+
+expect.extend(matchers);
+
+describe('Alert Dialog', () => {
+  it('Should render an alert dialog box', () => {
+    render(<AlertDialogExample />);
+    expect(screen.getByText('Submit Application')).toBeDefined();
+  });
+
+  it('Should only have one, labeled button when no secondary label or action is given', () => {
+    render(<AlertDialogForNotifying />);
+    expect(screen.getAllByRole('button')).toHaveLength(1);
+    expect(screen.getByRole('button')).toHaveTextContent('Okay');
+  });
+
+  it('Should have two, labeled buttons when a secondary label and action is given', () => {
+    render(<AlertDialogExample />);
+    const buttons = screen.getAllByRole('button');
+    expect(buttons).toHaveLength(2);
+    expect(buttons[0]).toHaveTextContent('Cancel');
+    expect(buttons[1]).toHaveTextContent('Submit');
+  });
+
+  it('Should have a destructive button style when the destructive prop is included', () => {
+    render(<AlertDialogDestructive />);
+    expect(screen.getByText('Delete')).toHaveStyle('background-color: color-background-destructive');
```

Comment:
Will this test fail if Alert Dialog is used with dark mode eventually?

Reply:
That's a good question- since what we're testing here is the token name not the value itself, this should pass even in dark mode (thanks to @richbachman for clarifying!)